pom.xml

```diff
@@ -621,7 +621,7 @@
                         <configuration>
                             <!-- examples of class excluding -->
                             <!--
- excludes>
+                                excludes>
```

Comment:
Unnecessary change. Can stay, but if rolled back, even better. and best if new line remove too.